src/unix/terminal.rs

```diff
@@ -6,7 +6,7 @@ use std::sync::atomic::{AtomicUsize, ATOMIC_USIZE_INIT, Ordering};
 use std::time::Duration;
 
 use libc::{
-    c_int, c_ushort, c_void,
+    c_int, c_ushort, c_ulong, c_void,
```

Comment:
This generates an `unused_imports` warning on non-freebsd platforms. You can instead add `use libc::c_ulong;` inside the freebsd version of `ioctl_wrapper`.